### PR TITLE
New sound stuff, fish hook predicate, missed persistent state uuid fix

### PR DESCRIPTION
**Pull request by liach**
Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

**Review by modmuss50 — APPROVED** [review]

**Review by Runemoro — APPROVED** [review]

File: mappings/net/minecraft/entity/projectile/FishingBobberEntity.mapping [hunk 1]
```diff
@@ -27,6 +28,11 @@ CLASS net/minecraft/class_1536 net/minecraft/entity/projectile/FishingBobberEnti
 		ARG 2 owner
 		ARG 3 lureLevel
 		ARG 4 luckOfTheSeaLevel
+	METHOD method_26086 isOpenOrWaterAround (Lnet/minecraft/class_2338;)Z
+		ARG 1 pos
+	METHOD method_26087 isOpenOrWater (Lnet/minecraft/class_2338;)Z
```
Comment (Runemoro) [inline mappings/net/minecraft/entity/projectile/FishingBobberEntity.mapping]: What does this mean?

Comment (liach) [reply mappings/net/minecraft/entity/projectile/FishingBobberEntity.mapping]: Returns whether this block is air or water/waterlogged.

Comment (liach) [reply mappings/net/minecraft/entity/projectile/FishingBobberEntity.mapping]: This is called by `isOpenOrWaterAround` in a blockpos stream in a 5 * 3 * 5 area to ensure in this area there is no artificial blocks. Looks like the fix for fishing machine.

File: mappings/net/minecraft/client/network/ClientPlayerEntity.mapping [hunk 2]
```diff
@@ -48,6 +48,7 @@ CLASS net/minecraft/class_746 net/minecraft/client/network/ClientPlayerEntity
 	METHOD method_22419 showsDeathScreen ()Z
 	METHOD method_22420 setShowsDeathScreen (Z)V
 		ARG 1 shouldShow
+	METHOD method_26269 getMoodPercentage ()F
```
Comment (Juuxel) [inline mappings/net/minecraft/client/network/ClientPlayerEntity.mapping:51]: Maybe clarify that this is for sounds?

Comment (liach) [reply mappings/net/minecraft/client/network/ClientPlayerEntity.mapping:51]: See [slicelime's intro vid for 20w12a](https://youtu.be/GowKC1iYz4c?t=570). On the debug screen, it says `Mood: 8%`. Should I clarify in javadocs instead?

Comment (liach) [reply mappings/net/minecraft/client/network/ClientPlayerEntity.mapping:51]: Added a piece of jd.

**Review by Juuxel — APPROVED** [review]

**Comment by modmuss50** [comment]
conflicts please.

**Comment by liach** [comment]
Wait a sec (need another rename)

**Pull request merged**